tests/test_modeling_common.py

```diff
@@ -4769,7 +4770,10 @@ def test_torch_compile(self):
         n_iter = 3
 
         tokenizer = AutoTokenizer.from_pretrained(ckpt)
-        model = AutoModelForCausalLM.from_pretrained(ckpt, torch_dtype=torch.float16).to(torch_device)
+        if self.is_encoder_decoder:
```

Comment:
for t5 are we only testing one of them in that case? (probably decoder?)

Reply:
we test the whole model as encoder-decoder, I added a test_ckpt as `t5-small`. Since the forward call includes call to encoder and to decoder